BioForm/api/urls.py

```diff
@@ -0,0 +1,19 @@
+from django.urls import path
+from rest_framework.routers import DefaultRouter
+from . import views
+from .views import BlogViewSet
+
+app_name = 'api'
+
+router = DefaultRouter()
+router.register('api/blogview',BlogViewSet, basename='blog')
```

Comment:
Keep the endpoint name pattern consistent.
```suggestion
router.register('api/blog/',BlogViewSet, basename='blog')
```

Reply:
Changed as per suggestion